docs/extend/settings.md

```diff
@@ -107,8 +113,62 @@ export default class HelloWorldSettingsModal extends SettingsModal {
 }
 ```
 
-Always remember to use [internationalizations](i18n.md)!
+Always remember to use [internationalizations](i18n.md) for any labels!
 
-Each `Form-group` should contain an individual input. The `bidi` property on the input corresponds to the setting on the Settings table that the input will modify. It will be automatically propagated with the current settings value, and it will change the value once "Save Settings" button is clicked.
+Each `Form-group` should contain an individual input. The `bidi` property on the input corresponds to the setting on the Settings table that the input will modify. It will be automatically propagated with the current settings value, and it will save the value to the database once "Save Settings" button is clicked.
 
 *Oh? What's this?* When you click "Save Settings" the (settings) [Saved](https://github.com/flarum/core/blob/master/src/Settings/Event/Saved.php) event is dispatched? How cool!
+
+### Accessing Settings
+
+All settings are available in the `admin` frontend via the `app.data.settings` global.
+However, this is not done in the `forum` frontend, as anyone can access it, and you wouldn't want to leak all your settings! (Seriously, that could be a very problematic data breach).
+
+Instead, if we want to use settings in the `forum` frontend, we'll need to serialize them and send them alongside the initial forum data payload.
+
+Currently, this can be done by listening in to the `Flarum\Api\Event\Serializing` event for `Flarum\Api\Serializer\ForumSerializer`. For example:
+
+**extend.php**
+
+```php
+use Flarum\Extend;
+use Flarum\Api\Event\Serializing;
+use Flarum\Api\Serializer\ForumSerializer;
+use Flarum\Settings\SettingsRepositoryInterface;
+
+use Acme\AddSettingsToPayload;
+
+
+return [
+   (new Extend\Event)->listen(Serializing::class, AddSettingsToPayload::class)
+]
+```
+
+```php
+namespace Acme;
+
+class AddSettingsToPayload
+{
+  /**
+  * @var SettingsRepositoryInterface
+  */
+  protected $settings;
+
+  /**
+    * @param SettingsRepositoryInterface $settings
```

Comment:
Again, I'd actually suggest removing the docblocks entirely. They take place and don't offer much help since the parameters are type-hinted already. We could keep it just for the class property above maybe.